modules/browser_object_autofill_popup.py

```diff
@@ -18,6 +20,14 @@ def press_doorhanger_save(self):
         with self.driver.context(self.driver.CONTEXT_CHROME):
             self.get_element("doorhanger-save-button").click()
 
+    def verify_autofill_displayed(self):
+        """Confirms that autofill popup has loaded"""
+        self.element_clickable("autofill-panel")
+
+    def verify_element_displayed(self, reference: Union[str, tuple, WebElement]):
```

Comment:
I think this method could be in `page_base` because it seems to be a pretty generic and doesn't just apply to popups, up to you